doc/network-zones.md

```diff
@@ -4,57 +4,149 @@ discourse: 12033,13128
 
 # How to configure network zones
 
-Network zones are used to hold DNS records for LXD networks.
+```{note}
+Network zones are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
-Each network can be related to up to 3 zones for:
+Network zones can be used to hold DNS records for LXD networks.
 
- - Forward DNS records
- - IPv4 reverse DNS records
- - IPv6 reverse DNS records
+If you are operating a LXD cluster with multiple instances across a large set of networks, you can use network zones to automatically maintain valid forward and reverse records for all your instances.
+Such DNS records are important not only for ease of access to the instances, but also to avoid hosted services getting flagged as potential spam due to lacking reverse DNS records.
 
-LXD will then be automatically managing forward and reverse records for all instances, network gateways and downstream network ports.
+Each network can be related to up to three zones for:
+
+- Forward DNS records
+- IPv4 reverse DNS records
+- IPv6 reverse DNS records
+
+LXD will then automatically manage forward and reverse records for all instances, network gateways and downstream network ports and serve those zones for zone transfer to the operator’s production DNS servers.
```

Comment:
Can we expand the example below to cover each of these 3 types: "instances, network gateways and downstream network ports", so we can document the format that these records will be created, as well as what a "downstream network port" is (I don't know :))